buildSrc/subprojects/parser/src/main/java/org/apache/calcite/buildtools/parser/DialectGenerate.java

```diff
@@ -120,10 +125,152 @@ public void unparseReservedKeywords(ExtractedData extractedData) {
       tokens.add(tokenBuilder.toString());
     }
     stringBuilder.append(String.join("| ", tokens));
-    stringBuilder.append("}\n");
+    stringBuilder.append("}");
```

Comment:
This is more of an open question. Is there any side effect introduced in the change here to not add the next line character in the resulted generated file?

Reply:
Good question. I made this change to stay consistent with how the tokens/functions are unparsed in DialectTraverser here: https://github.com/googleinterns/calcite/blob/733e88cb24142b26dd8e89d5e7bd9fbe0bb793b3/buildSrc/subprojects/parser/src/main/java/org/apache/calcite/buildtools/parser/DialectTraverser.java#L94

As you can see the newline is being added there so adding it here would result in a double new line.